imports/api/genes/addInterproscan.js

```diff
@@ -1,206 +1,56 @@
-import { Meteor } from 'meteor/meteor';
+import jobQueue, { Job } from '/imports/api/jobqueue/jobqueue.js';
 import { ValidatedMethod } from 'meteor/mdg:validated-method';
-import { Roles } from 'meteor/alanning:roles';
-
-import SimpleSchema from 'simpl-schema';
-import Papa from 'papaparse';
-import fs from 'fs';
-
-// import { chunk } from 'lodash';
-
 import { Genes } from '/imports/api/genes/geneCollection.js';
-// import { Interpro } from '/imports/api/genes/interpro_collection.js';
 import logger from '/imports/api/util/logger.js';
+import { Roles } from 'meteor/alanning:roles';
+import SimpleSchema from 'simpl-schema';
+import { Meteor } from 'meteor/meteor';
 
-import {
-  parseAttributeString, debugParseAttributeString, DBXREF_REGEX,
-} from '/imports/api/util/util.js';
-
-const logAttributeError = ({
-  lineNumber, line, attributeString, error,
-}) => {
-  logger.warn(`Error line ${lineNumber}`);
-  logger.warn(line.join('\t'));
-  logger.warn(attributeString);
-  logger.warn(debugParseAttributeString(attributeString));
-  throw new Meteor.Error(error);
-};
-
-class InterproscanProcessor {
+class InitializeGenes {
```

Comment:
I'd rename this to something more explicit (InterproscanProcessor like before? InterproscanParser? something like this)